awxkit/awxkit/api/pages/credentials.py

```diff
@@ -287,7 +287,7 @@ def create_payload(
 
         credential_type, organization, user, team = filter_by_class(
             (credential_type, CredentialType), (organization, Organization), (user, User), (team, Team))
-        if not any((user, team, organization)):
+        if not all((user, team, organization)):
```

Comment:
I wonder if you want to check for each one individually.

If you have a user, get the org from it.

If you have a team, get the org from it,

If you have an org...use it.